### ISSUE TITLE
Norconex Solr Committer - Corrupts Collection

### ISSUE THREAD

**Issue by RaviVelaga2018** [Norconex/collector-http#499]
I have Apache Solr7.2 installed in SolrCloud mode. I have setup 4 collections, to index html pages, images & videos in separate collection.  I was using Norconex to index the content from the web site. I setup multiple Norconex HTTP Crawlers.  One Norconex crawler to index html page to index to Solr html collection and another one for images etc...  One I ran one crawler at a time everything is working fine but if I start crawler two crawlers at the same time and indexing the Solr collections are getting messed up of images will start showing in html collection.  Can you please know is there any limitation on crawlers that need to be run at a time.

**Comment by essiembre** [comment]
There is no limitation but it is likely a case of overlapping directories.  In each of your Solr Committer configuration, make sure to specify different path under ``<queueDir>...</queueDir>``.

This is usually the culprit, but make sure you do not have other overlapping paths in your config.

**Comment by RaviVelaga2018** [comment]
Thanks for pointing me to the right direction.  You are absolutely right, <queueDir> is the issue. 

Just to give little bit of explanation in case if some one in the same situation as me.  Initially I didn't include in the <queueDir> in my crawler configuration and the crawler took the default configuration is using the <Norconex Install>/committer-queue/ directory.  When I ran only one crawler at a time no issue, since the <queueDir> has to deal with one crawler, but when I ran two crawlers at the same time both are using <queueDir> and grabbing whatever in the <queueDir>directory and uploading it to both collections, making it completely undesirable results.

Thanks again for your help, Essiembre.
